.github/CODEOWNERS

```diff
@@ -197,7 +197,7 @@ processor/sumologicprocessor/                                       @open-teleme
 processor/tailsamplingprocessor/                                    @open-telemetry/collector-contrib-approvers @jpkrohling
 processor/transformprocessor/                                       @open-telemetry/collector-contrib-approvers @TylerHelmuth @kentquirk @bogdandrutu @evan-bradley
 
-receiver/activedirectorydsreceiver/                                 @open-telemetry/collector-contrib-approvers @djaglowski @BinaryFissionGames
+receiver/activedirectorydsreceiver/                                 @open-telemetry/collector-contrib-approvers @djaglowski @BinaryFissionGames @pjanotti
```

Comment:
@djaglowski would like to be removed from this one and the IIS one?

Reply:
Yes, I'll happily drop from those. Thank you for taking them on.

Reply:
[1f5b222](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33695/commits/1f5b222295ed89af3ebb3fffa86924771629a885)